modules/loaders-common/src/main/scala/com/snowplowanalytics/snowplow/loaders/transform/NonAtomicFields.scala

```diff
@@ -47,18 +50,41 @@ object NonAtomicFields {
 
   def resolveTypes[F[_]: Sync: RegistryLookup](
     resolver: Resolver[F],
-    entities: Map[TabledEntity, Set[SchemaSubVersion]]
+    entities: Map[TabledEntity, Set[SchemaSubVersion]],
+    entitiesToSkip: List[SchemaCriterion]
   ): F[Result] =
     entities.toList
       .traverse { case (tabledEntity, subVersions) =>
         SchemaProvider
           .fetchSchemasWithSameModel(resolver, TabledEntity.toSchemaKey(tabledEntity, subVersions.max))
-          .map(TypedTabledEntity.build(tabledEntity, subVersions, _))
+          .map(schemas => filterOutMatchingSubversions(entitiesToSkip, schemas))
```

Comment:
The reason for this feature, is because it gives us flexibility to ignore errors that would otherwise cause failed events or loading errors.

Here's an example.....  imagine a user deletes a schema from their Iglu Server.  The trackers are using that schema to add a self-describing context to all events.  The user wants to load the events, simply ignoring that context.

But currently that wouldn't work!  Because `SchemaProvider.fetchSchemasWithSameModel` would fail first, before you `filterOutMatchingSubversions`.  Is it possible to somehow move the filter up a bit earlier?  So we filter first, and then afterwards fetch from Iglu?